packages/web/src/components/modal-form/ModalForm.tsx

```diff
@@ -0,0 +1,22 @@
+import cn from 'classnames'
+import { Form, FormikFormProps } from 'formik'
+
+import styles from './ModalForm.module.css'
+
+/**
+ * Styles a Formik Form properly to allow it to parent the contents of a modal
+ * while maintaining the normal behaviors of ModalHeader, ModalFooter, and ModalContent.
+ * Can't be moved to Stems or else Formik contexts are mismatched between versions if used in Web.
+ * Must be used inside the <Formik> context.
+ */
+export const ModalForm = ({
```

Comment:
@dylanjeffers this ended up being easier than I expected, so maybe this extra component is kinda overkill. LMK what you think